x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.tsx

```diff
@@ -53,25 +67,32 @@ const columns: Array<EuiTableFieldDataColumnType<CrawlEvent>> = [
         defaultMessage: 'Status',
       }
     ),
-    align: 'right',
-    render: (status: CrawlEvent['status'], event: CrawlEvent) => (
-      <>
-        {event.stage === 'process' && (
-          <EuiIconTip
-            aria-label="Process crawl"
-            size="m"
-            type="iInCircle"
-            color="primary"
-            position="top"
-            content="Re-applied crawl rules"
-          />
-        )}
-        {readableCrawlerStatuses[status]}
-      </>
-    ),
+    render: (status: CrawlEvent['status']) => readableCrawlerStatuses[status],
   },
 ];
 
+export const CrawlEventTypeBadge: React.FC<{ event: CrawlEvent }> = ({ event }) => {
+  if (event.stage === 'process') {
+    return (
+      <EuiBadge color="hollow">
+        {i18n.translate(
+          'xpack.enterpriseSearch.appSearch.crawler.crawlTypeOptions.reAppliedCrawlRules',
+          {
+            defaultMessage: 'Re-applied crawl rules',
+          }
+        )}
+      </EuiBadge>
+    );
+  }
+  if (event.type === CrawlType.Full) {
+    return <EuiBadge>{readableCrawlTypes[CrawlType.Full]}</EuiBadge>;
+  }
+  if (event.type === CrawlType.Partial) {
+    return <EuiBadge color="hollow">{readableCrawlTypes[CrawlType.Partial]}</EuiBadge>;
+  }
+  return null;
+};
+
```

Comment:
I'm not going to shut down your PR for this, but I think we should be pulling stuff like this out to its own file. Exporting two components from a single file gets messy. I also have a tendency to find code for components with "cmd + p" in VS Code and finding a corresponding file name.

If this component were being used as a "private" method, not exported, and tested via the primary component's interface than I'd say it's fine inline in the file. But since we're exporting it and testing it separately, then I think it should have its own file.

Reply:
I'd still consider it a private method even though I'm testing it directly, but I moved it out to its own file, I'll keep this in mind going forward!